lib/util.js

```diff
@@ -2998,3 +2998,26 @@ util.estimateCores = function(options, callback) {
     }, 0);
   }
 };
+
+util.compareDN = function(l, r) {
```

Comment:
This is currently just used internally, but some doc strings would be good.  "l" may not be the best choice for a param.  Looks a lot like "I" in many fonts.  I'm guessing this is left vs right.  Could just as easily be a b or x0 x1, etc.

Reply:
Documentation added.  Parameters renamed to ```dn1``` and ```dn2```.